.github/workflows/deploy_preview.yml

```diff
@@ -133,3 +135,12 @@ jobs:
           echo "Preview URL: ${{ steps.netlify_deploy_beta_preview.outputs.NETLIFY_PREVIEW_URL }}" >> netlify.txt
           echo "Logs: ${{ steps.netlify_deploy_beta_preview.outputs.NETLIFY_LOGS_URL }}" >> netlify.txt
           gh pr comment ${{ github.event.pull_request.number }} --body-file netlify.txt || true
+
+      - uses: chrnorm/deployment-action@v2
+        name: Create GitHub deployment
+        id: deployment
+        with:
+          token: '${{ github.token }}'
+          environment-url: ${{ steps.netlify_deploy_testnet_preview.outputs.NETLIFY_PREVIEW_URL }}
+          environment: Preview
+          initial-status: success
```

Comment:
Think we can:
1. Rename it to `Create GitHub Preview deployment` so we won't mix it with beta.
2. Move to line 92, right after `Deploy testnet preview to Netlify` so we can run tests earlier.